### PR TITLE
expose more TLS config options

### PR DESCRIPTION
**Pull request by urso**
Expose all TLS config options used by golang tls clients via config
file.

TLS config parsing requires go1.5 to build (#123). Requirement due to TLS cipher suites added in go1.5.

### CODE REVIEW

**Comment by urso** [comment]
Documentation will be done in another PR (see issue #100)

File: outputs/tls.go [hunk 1]
```diff
@@ -68,14 +94,102 @@ func LoadTLSConfig(config MothershipConfig) (*tls.Config, error) {
...
+	case "1.2":
+		return tls.VersionTLS12, nil
+	default:
+		return 0, ErrInvalidTLSVersion
+	}
+}
+
+func parseTLSCipherSuites(names []string) ([]uint16, error) {
+	suites := map[string]uint16{
+		"RSA-RC4-128-SHA":                tls.TLS_RSA_WITH_RC4_128_SHA,
```
Comment (jordansissel) [inline outputs/tls.go]: RC4 is not recommended; should we disable it?

Comment (urso) [inline outputs/tls.go]: same goes for AES. I just keep it in for now to be compatible + expose all known (to golang) config options. Easier to maintain list of known codecs vs. list of recommended codecs...

"Recommendations" we should put into doc?

Optionally will we can add warning if not recommended codec is used.

If cipher suite is not configured, golang will use internal default list with recommended order (Have stdlib authors maintain this list).

Comment (ruflin) [inline outputs/tls.go]: I added a note to https://github.com/elastic/libbeat/issues/100 to document it.

Comment (jordansissel) [inline outputs/tls.go]: +1

**Pull request merged**